migrations/migrations/20191206223821-create-albums.js

```diff
@@ -0,0 +1,33 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.createTable('Albums', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      userId: {
```

Comment:
remember use `snake_case` here, this is after you set `underscored` option in the model